demo/EXPLOR3R/auto-drive.py

```diff
@@ -48,6 +48,8 @@
 ir = InfraredSensor(); assert ir.connected
 ts = TouchSensor();    assert ts.connected
 
+print('Robot Starting')
```

Comment:
Is there a demo somewhere on how to vim a .sh file and chmod + x it so it can be launched? Or is there some other way to make these scripts executable on the brickman?